xbmc/input/ButtonTranslator.cpp

```diff
@@ -324,7 +324,7 @@ static const ActionMapping windows[] =
     { "pvrchannelscan"           , WINDOW_DIALOG_PVR_CHANNEL_SCAN },
     { "pvrupdateprogress"        , WINDOW_DIALOG_PVR_UPDATE_PROGRESS },
     { "pvrosdchannels"           , WINDOW_DIALOG_PVR_OSD_CHANNELS },
-    { "pvrosdguide"              , WINDOW_DIALOG_PVR_OSD_GUIDE },
```

Comment:
I think it would be nice if we'd keep this ID in addition to the new one for backwards compat reasons to not kill all custom user keymaps. We could add a comment to remove it in like 2 versions from now.